src/shared/components/simpleTable/SimpleTable.tsx

```diff
@@ -0,0 +1,102 @@
+import * as React from 'react';
+import './styles.scss';
+import {observable, computed, action} from "mobx";
+import {observer} from "mobx-react";
+
+export type Column = {
+    name:string;
+    sortable?:boolean;
+};
+
+export type Row = { [columnName:string]:JSX.Element };
+
+
+type SimpleTableProps = {
+    columns:Column[];
+    rows:Row[];
+    onSort?:(name:string, sortAscending:boolean)=>void;
+};
+
+class SimpleTableStore {
```

Comment:
That makes sense. Then we can make it onHeaderClick which is more general and have the parent handle all the logic there. So this component is really just a visualizing shell.